diff --git a/dist/index.mjs b/dist/index.mjs
index b8c96da82fc2f983cfdbe2d52764ffaffea8b520..1ee57c546e3173670676f56dba50c9a085fc2e3e 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -2448,7 +2448,10 @@ async function loadNuxtModuleInstance(nuxtModule, nuxt = useNuxt()) {
     for (const path of paths) {
       const src = await resolvePath(path);
       try {
-        nuxtModule = await importModule(src, nuxt.options.modulesDir).catch(() => null) ?? requireModule(src, { paths: nuxt.options.modulesDir });
+        nuxtModule = await importModule(src, nuxt.options.modulesDir).catch((e) => {
+          console.log(src, e, nuxt.options.modulesDir)
+          return null
+        }) ?? requireModule(src, { paths: nuxt.options.modulesDir });
         const moduleMetadataPath = join(dirname(src), "module.json");
         if (existsSync(moduleMetadataPath)) {
           buildTimeModuleMeta = JSON.parse(await promises.readFile(moduleMetadataPath, "utf-8"));
