diff --git a/dist/index.mjs b/dist/index.mjs
index a9541e0f283ee7d12a82962d76916cfd09b95d16..68df7f600164228d50f033bf35771a60ee78914b 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1892,6 +1892,11 @@ function packageResolve(specifier, base, conditions) {
     // Cross-platform root check.
   } while (packageJsonPath.length !== lastPath.length)
 
+  try {
+    fileURLToPath$1(base)
+  } catch (e) {
+    console.log({e, base: base.toString()})
+  }
   throw new ERR_MODULE_NOT_FOUND(packageName, fileURLToPath$1(base), false)
 }
 
